src/content/fr/2019/resource-hints.md

```diff
@@ -0,0 +1,250 @@
+
+### `preconnect`
+
+[`preconnect`](https://web.dev/uses-rel-preconnect) initie une connexion anticipée, y compris la résolution DNS, la poignée de main TCP et la négociation TLS. Cette fonctionnalité est utile pour établir une connexion avec une tierce partie. Les utilisations de `preconnect` sont très similaires à celles de `dns-prefetch`, mais `preconnect` est moins bien supporté par les navigateurs. Cependant, si vous n'avez pas besoin du support d'IE 11, `preconnect` est probablement un meilleur choix.
+
+### `preload`
+
+[`preload`](https://medium.com/reloading/preload-prefetch-and-priorities-in-chrome-776165961bbf) initie une requête au plus tôt. Il permet de charger des ressources importantes plutôt que d'attendre que l'analyseur les découvre plus tardivement. Par exemple, si une image importante ne peut être découverte qu'une fois que le navigateur a reçu et analysé la feuille de style, il est peut-être judicieux de précharger l'image.
+
+### `prefetch`
+
+[`prefetch`](https://calendar.perfplanet.com/2018/all-about-prefetching/) lance une requête de faible priorité. C'est utile pour charger les ressources qui seront utilisées lors du chargement de la page suivante (plutôt que de la page actuelle). Une utilisation courante de `prefetch` est le chargement de ressources dont l'application "prédit" qu'elles seront utilisées lors du chargement de la page suivante. Ces prédictions peuvent être basées sur des signaux tels que le mouvement de la souris de l'utilisateur ou des scénarios&#8239;/&#8239;parcours utilisateurs courants.
+
+## Syntaxe
+
+97 % des usages d'indices de ressources sont spécifiés par le tag [`<link>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link). Par exemple, `<link rel="prefetch" href="shopping-cart.js">`.
+
+Seuls 3 % des indices de ressources proviennent [d'en-têtes HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link) permettant de les spécifier. Par exemple, `Link: <https://example.com/shopping-cart.js>; rel=prefetch`.
+
+Comme l'utilisation des indices de ressources dans les en-têtes HTTP est très faible, le reste de ce chapitre se concentrera uniquement sur l'analyse de l'utilisation des indices de ressources passant par la balise `<link>`. Cependant, il convient de noter que dans les années à venir, l'utilisation des indices de ressources dans les en-têtes HTTP pourrait augmenter avec l'adoption de [HTTP/2 Push](./http2#http2-push). En effet, HTTP/2 Push réutilise l'en-tête HTTP de préchargement `Link` comme un signal permettant de pousser des ressources.
+
+## Les indices de ressources
+
+<aside class="note">Note: il n'y a pas de différence notable entre l'usage des indices de ressources sur les pages web destinées aux ordinateurs ou périphériques mobiles. C'est pourquoi, par souci de concision, ce chapitre n'inclut que les statistiques relatives aux mobiles.</aside>
+
+<figure>
+  <table>
+    <tr>
+     <th>Resource Hint</th>
```

Comment:
We should add `scope="col"` to `th`s

Reply:
As it's not relative to this translation, maybe create a separate issue?

Reply:
Or better yet an issue and a PR to resolve 😉

The default, if not specified is `scope="auto"` which I understand should be the same as `col` in this case and this is more useful for mixed headers? But not really used this so could be wrong.

Either way I agree it's separate to this translation and should be raised as separate issue if you think we need to look at it.

Reply:
Issue created: https://github.com/HTTPArchive/almanac.httparchive.org/issues/657